src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<blockquote>... the right solution<code>.</code></blockquote><p> </p><p>Also, fun fact: Plotly allow to zoom and find the correct value rather accurately :)</p><p>Not sure it's worth mentioning, though...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Changed in dfe233f7. Added a remark for Plotly